EIPS/eip-3554.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 3554
+title: Difficulty Bomb Delay to December 1st 2021
+author: James Hancock (@madeoftin)
+discussions-to: https://github.com/ethereum/EIPs/issues/3554
```

Comment:
This has to be a link to somewhere other than the PR itself.  https://ethereum-magicians.org/ thread is preferred, a GitHub issue in this repository can be used if you are unable/unwilling to create a magicians thread.

Reply:
Given this supersedes 3238, can we use the same discussions-to link, i.e. https://github.com/ethereum/EIPs/issues/3239?

Reply:
@lightclient weird - that issue redirects to this page.

Reply:
Ohh I see, that is the PR number and so github just redirects it to PR even if it says its an issue. In that case, we haven't allowed discussion links to be PRs for a while.

Reply:
`/issues/` and `/pulls/` are basically aliases of each other on GitHub.  All that matters is the number.

Reply:
I would *prefer* a new issue just to reduce confusion.  Otherwise you can end up with two people talking about different things in the discussion.